include/sketch.h

```diff
@@ -72,7 +72,7 @@ class Sketch {
    * @return               The number of samples
    */
   static size_t calc_cc_samples(node_id_t num_vertices, double f) {
-    return ceil(f * log2(num_vertices) / num_samples_div);
+    return std::max(size_t(18), (size_t) ceil(f * log2(num_vertices) / num_samples_div));
```

Comment:
I want to make sure I fully understand this change before I approve it

Reply:
This is enforcing that every connectivity sketch contains at least 18 columns. I made this change because at the small sizes our unit tests are running our failure rate is a bit too high. As the number of vertices increases the failure rate should decrease.